src/styles/index.scss

```diff
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import 'main';
```

Comment:
Ensure that the 'main' SCSS file exists in the same directory or provide the correct relative path if it's located elsewhere. Also, remember to compile the SCSS into CSS before using it in your HTML.